examples/advanced_tensorflow/run.sh

```diff
@@ -1,10 +1,16 @@
 #!/bin/bash
 
+./certificates/generate.sh
+
 echo "Starting server"
+
 python server.py &
 sleep 3  # Sleep for 3s to give the server enough time to start
 
-for i in `seq 0 9`; do
+# Ensure that the Keras dataset used in client.py is already cached.
+python -c "import tensorflow as tf; tf.keras.datasets.cifar10.load_data()"
+
+for i in `seq 0 10`; do
```

Comment:
Shouldn't this remain unchanged?
```suggestion
for i in `seq 0 9`; do
```